pkg/action/install.go

```diff
@@ -405,6 +407,7 @@ func (i *Install) performInstall(c chan<- resultMessage, rel *release.Release, t
 		rel.SetStatus(release.StatusDeployed, "Install complete")
 	}
 
+	rel.SetReleaseTag(i.ReleaseTag)
```

Comment:
Is there a reason this needs to be a function? Can the public property just be set to be equal? `SetStatus` exists because 2 things are set at the same time. It's the only property updated with a function.

Reply:
no problem i can switch it to that i was just following the pattern for SetStatus